chroma_core/models/copytool.py

```diff
@@ -387,7 +387,7 @@ def run(self, kwargs):
             bin_path=copytool.bin_path,
             archive_number=copytool.archive,
             filesystem=filesystem.name,
-            mountpoint=client_mount.mountpoint,
+            mountpoint=client_mount.mountpoints[0],
```

Comment:
Are we guaranteed that mountpoints isn't empty?

Reply:
Yes, the invariant is the same as before, now it's just a list.